src/message/messagesActions.js

```diff
@@ -73,27 +81,31 @@ export const fetchMessages = (
   narrow,
   useFirstUnread = false,
 ) =>
-  async (dispatch) => {
+  async dispatch => {
     if (numBefore < 0 || numAfter < 0) {
       throw Error('numBefore and numAfter must >= 0');
     }
 
-    dispatch(messageFetchStart(
-      narrow,
-      {
-        ...numBefore ? { older: true } : {},
-        ...numAfter ? { newer: true } : {},
-      },
-    ));
+    dispatch(
+      messageFetchStart(narrow, {
+        ...(numBefore ? { older: true } : {}),
+        ...(numAfter ? { newer: true } : {}),
+      }),
+    );
     dispatch(backgroundFetchMessages(auth, anchor, numBefore, numAfter, narrow, useFirstUnread));
   };
 
-export const updateMessageFlags = (auth, messageIds, op, flag) =>
-  async (dispatch) => {
-    await messagesFlags(auth, messageIds, op, flag);
```

Comment:
It is extracted in the queueMarkAsRead. This is just code that was forgotten to be removed.